source/common/access_log/access_log_formatter.cc

```diff
@@ -169,10 +169,10 @@ void AccessLogFormatParser::parseCommand(const std::string& token, const size_t
       throw EnvoyException(fmt::format("Incorrect position of ')' in token: {}", token));
     }
 
-    std::string length_str = token.substr(end_request + 2);
+    const auto length_str = absl::string_view(token).substr(end_request + 2);
```

Comment:
add a TODO above to make parameter `token` as string_view? (And I think with that we can omit `start` parameter as well.)